tutorials/docs/workflow.md

```diff
@@ -0,0 +1,96 @@
+# WhizzML: Training and Predicting Workflows
+
+## The detailed Training Workflow
```

Comment:
it would perhaps be useful to show either a complete whizzml source code snippet, or individual snippets in each section, no?

Reply:
done